Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponentInterface.h

```diff
@@ -84,6 +85,21 @@ namespace AzToolsFramework
                 AZStd::vector<AZStd::unique_ptr<Instance>>&& instancesToConsume, AZ::IO::PathView filePath,
                 AZStd::unique_ptr<AZ::Entity> containerEntity = nullptr, InstanceOptionalReference parent = AZStd::nullopt,
                 bool shouldCreateLinks = true) = 0;
+
+            //! Creates a prefab instance using the entities and nested instances provided.
+            //! @param entities The map of entities to add to the instance. Map ensures insertion order.
+            //! @param nestedInstances The instances to add to the newly created instance.
+            //! @param filePath The path at which the prefab should be created.
+            //! @param containerEntity The container entity for the new instance.
+            //! @param parent The parent instance of the newly created instance.
+            //! @return The new instance created.
+            virtual AZStd::unique_ptr<Instance> CreatePrefab(
+                const AZStd::map<EntityAlias, AZ::Entity*>& entities,
+                AZStd::vector<AZStd::unique_ptr<Instance>>&& nestedInstances,
```

Comment:
Is 'nestedInstances' the same as 'instancesToConsume' in the other 'CreatePrefab' function?

Reply:
Yes. I think it's a better name than 'consume'. I'll change in other places too.